src/test/java/com/epam/tc/hw3/ex1/FirstExercise.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.BaseClass;
+import com.epam.tc.hw3.pages.IndexPage;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+
+public class FirstExercise extends BaseClass {
+
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void firstTest() {
```

Comment:
Please rename test method with some meaningful name.
Test methods should have a name in accordance with checking functionality

Reply:
Renamed